cmake/Modules/ObsHelpers.cmake

```diff
@@ -312,12 +312,6 @@ endfunction()
 macro(find_qt)
   set(multiValueArgs COMPONENTS COMPONENTS_WIN COMPONENTS_MAC COMPONENTS_LINUX)
   cmake_parse_arguments(FIND_QT "" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
-  set(QT_NO_CREATE_VERSIONLESS_TARGETS ON)
```

Comment:
Why is this removed?

Reply:
Read the PR description which explain why.

Reply:
The underlying issue seems to be that the properties view seems to run `find_qt` twice in the same scope, which was never supported (we never had that issue so far), so the question is why Qt is looked for twice (otherwise it wouldn't run into the collision). That should be investigated/fixed first.

Reply:
The root cause for the issue is that if you add the subdirectory _after_ the `find_qt` call, then the already found Qt is inherited by the subdirectory (and thus you run into the collision).

If you add the subdirectory _before_ `find_qt`, then the targets stay in the subdirectory context and are not populated up. This would apply for every instance where the shared library is added after `find_qt`.

This obviously only applies to legacy CMake files and websockets/browser which have not been upgraded yet.

Reply:
It works, thank you pushed the changes.